Library/Homebrew/rubocops/shared/desc_helper.rb

```diff
@@ -0,0 +1,101 @@
+# frozen_string_literal: true
+
+require "rubocops/shared/string_content"
+require "rubocops/shared/helper_functions"
+
+# TODO: move the micro shared modules above into this one?
```

Comment:
This comment should be removed; I didn't do it because I was unsure about the code organization.

Reply:
Just remove the comment.

Reply:
Done :) (for the record, as we discussed [elsewhere](https://github.com/Homebrew/brew/pull/8137#discussion_r467477926), `string_content` was merged into `helper_functions`.)